PayPal.xcodeproj/xcshareddata/xcschemes/UnitTests.xcscheme

```diff
@@ -8,7 +8,7 @@
       <BuildActionEntries>
          <BuildActionEntry
             buildForTesting = "YES"
-            buildForRunning = "NO"
+            buildForRunning = "YES"
```

Comment:
What is this needed for?

Reply:
To support `run_tests`, per https://github.com/paypal/iOS-SDK/pull/4#discussion_r686127654

Reply:
Reverted